.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "mix"
+    directory: "/"
+    open-pull-requests-limit: 30
+    schedule:
+      interval: "daily"
+      time: "03:37" # UTC
```

Comment:
🎨 I'm comfortable in not having a specific time.
So I rather remove this line 'cause I don't have a good explanation why it should be `03:37`.